project 3 guidelines

weston-bailey opened the pull request:
[Summary of changes goes here]

* Change 1
* Change 2

bhague1281 commented:
Approved

[![Approved with PullApprove](https://img.shields.io/badge/pullapprove-approved-brightgreen.svg)](https://pullapprove.com/WDI-SEA/notes/pull-request/59/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=WDI-SEA/notes)

TaylorDarneille committed:
updated to remove dino blog

It would be too much to refactor the original dino blog to functional components

TaylorDarneille committed:
Update lab-router.md

weston-bailey committed:
updates mongo install notes

weston-bailey committed:
updates mac os installfest unit 3

weston-bailey committed:
updates p3 reqs
